test/selectors/channels.test.js

```diff
@@ -541,13 +541,19 @@ describe('Selectors.Channels', () => {
         const fromOriginalState = Selectors.getSortedUnreadChannelIds(testState);
         const fromModifiedState = Selectors.getSortedUnreadChannelIds(modifiedState);
         const fromMentionState = Selectors.getSortedUnreadChannelIds(mentionState);
+        console.log('chan2.id', chan2.id, chan2.display_name, chan2.name);
+        console.log('channel8.id', channel8.id, channel8.display_name, channel8.name);
```

Comment:
Forgotten console statements?

Reply:
Oh right, I forgot to remove it.  Nice catch, thanks!  Now updated.